source/release-notes/v2.0-release-notes.rst

```diff
@@ -0,0 +1,51 @@
+.. _v2.0-release-notes:
+
+v2.0 Release Notes
+==================
+
+Highlights in 2.0:
+
+- `Pinning apps to the dashboard`_
+- `Customizable layouts in the dashboard`_
+- `Adding metadata to app manifests`_
+- `Shell app now has themes`_
+- `Configurations in an ondemand.d directory`_
+- `ActiveJobs now a part of the dashboard`_
```

Comment:
Several core apps now part of the dashboard app

Then we can mention files, file editor, and active jobs

Reply:
Yea, file editor isn't really user facing (you wouldn't know it's different) and files could probably use it's own section, which I just forgot.

Reply:
I'ave added the files app as a whole new section. I figured for activejobs it's the image of the new app but also all the breaking changes. With the files app it's it's whole thing that may deserve more detail.